### PR TITLE
Add Straton Weather detection

### PR DESCRIPTION
**Pull request by StratonPro**
Hello,

Straton Weather detection

https://straton.pro

web app

examples:
https://app.straton.pro
http://www.xalazi.gr/
https://marousakis.gr/
https://maps.marousakis.gr/
https://ivm.straton.pro/

Thank You!

### CODE REVIEW

File: src/technologies/s.json [hunk 1]
```diff
@@ -16,6 +16,31 @@
     "scriptSrc": "\\.salesmanago\\.com/",
     "website": "https://www.salesmanago.com"
   },
+   "Straton Weather": {
+      "cats": [
+        5
+      ],
+      "cookies": {
+      "cookie_name": "STRATON"
```
Comment (AliasIO) [inline src/technologies/s.json:24]: Should this be `"STRATON": ""`?

File: src/technologies/s.json [hunk 2]
```diff
@@ -16,6 +16,31 @@
...
+        "X-Powered-By": "Straton(?:/([\\d.]+))?\\;version:\\1"
+      },
+      "meta": {
+        "generator": "Straton",
+        "keywords": "Straton"
+      },
+      "description": "Straton Weather Agency, We help entrepreneurs and businesses achieve optimal weather forecasting success!",
+      "html": "<!--[^>]+straton",
+      "dom": "a[href*='.straton.pro'][target='_blank']",
+      "icon": "Straton.svg",
```
Comment (AliasIO) [inline src/technologies/s.json:36]: Please remove the PNG icon file.

File: src/technologies/s.json [hunk 3]
```diff
@@ -16,6 +16,31 @@
...
+      "implies": [
+        "PHP"
+      ],
+      "pricing": [ "low" ],
+      "saas": true,
+      "website": "https://app.straton.pro"
```
Comment (AliasIO) [inline src/technologies/s.json:42]: Should this be `https://straton.pro`?

File: src/technologies/s.json [hunk 4]
```diff
@@ -16,6 +16,31 @@
...
+        "keywords": "Straton"
+      },
+      "description": "Straton Weather Agency, We help entrepreneurs and businesses achieve optimal weather forecasting success!",
+      "html": "<!--[^>]+straton",
+      "dom": "a[href*='.straton.pro'][target='_blank']",
+      "icon": "Straton.svg",
+      "implies": [
+        "PHP"
+      ],
+      "pricing": [ "low" ],
```
Comment (AliasIO) [inline src/technologies/s.json:40]: Please change to `"pricing": [ "recurring", "low" ],`

File: src/technologies/s.json (hunk 1 above)
Comment (StratonPro) [reply src/technologies/s.json]: Fix ok

      "cookies": {
      "Straton": ""
      },

File: src/technologies/s.json (hunk 3 above)
Comment (StratonPro) [reply src/technologies/s.json]: Fix OK

      "website": "https://straton.pro"

File: src/technologies/s.json (hunk 4 above)
Comment (StratonPro) [reply src/technologies/s.json]: Fix OK

      "pricing": [ "recurring", "low" ],

File: src/technologies/s.json (hunk 2 above)
Comment (StratonPro) [reply src/technologies/s.json:36]: Fix ok

update new upload SVG icon

**Pull request closed without merging**